### PR TITLE
chore: update roles

### PR DESCRIPTION
**Pull request by boojack**

### CODE REVIEW

File: content/docs/en/concepts/roles-and-permissions.md [hunk 1]
```diff
@@ -19,36 +19,36 @@ Workspace role and project role are **disjoint**, which means a Workspace Owner
...
+| Edit environment                             |           | ✔️  | ✔️    |
+| Reorder environment                          |           | ✔️  | ✔️    |
+| Archive environment                          |           | ✔️  | ✔️    |
+| Add instance                                 |           | ✔️  | ✔️    |
+| Edit instance                                |           | ✔️  | ✔️    |
+| Archive instance                             |           | ✔️  | ✔️    |
+| Sync instance schema                         |           | ✔️  | ✔️    |
+| Re-assign issue                              |           | ✔️  | ✔️    |
+| Become issue assignee                        |           | ✔️  | ✔️    |
+| Edit external SQL console                    |           | ✔️  | ✔️    |
```
Comment (tianzhou) [inline content/docs/en/concepts/roles-and-permissions.md:46]: This feature has been removed.

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/docs/en/concepts/roles-and-permissions.md (hunk 1 above)
Comment (boojack) [reply content/docs/en/concepts/roles-and-permissions.md]: Removed

**Comment by boojack** [comment]
PTAL

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**